src/main/kotlin/no/nav/klage/oppgave/api/view/KlagebehandlingEditableFieldsInput.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.klage.oppgave.api.view
+
+data class KlagebehandlingEditableFieldsInput(
+    val klagebehandlingVersjon: Long,
+    val internVurdering: String?,
+    val sendTilbakemelding: Boolean?,
+    val tilbakemelding: String?,
```

Comment:
Vi har ikke hørt om `sendTilbakemelding` og `tilbakemelding` før.
@svenanders @mahval

Reply:
Å nei, mulig de ikke er i gui (ennå), på samme måte som med eøs og raadfoertMedLege. Da kan jeg isåfall ta dem bort. Det er felter som var med tidlig i prosessen og skissene, hvor man velger om man skal sende tilbakemelding til førsteinstans, og isåfall hva den er. Er alt det borte fra dagens skisser?

Reply:
Da har jeg tatt de bort (i første omgang bare kommentert dem ut..)

Reply:
Har tatt de bort nå